Keyboard Inputs Unresponsive, Mouse Still Functional

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

During use, suddenly all keyboard inputs stopped responding, but mouse inputs were still functional.

### Environment

Zed: v0.146.5 (Zed)
OS: Linux X11 arch unknown
Memory: 31.3 GiB
Architecture: x86_64
GPU: NVIDIA GeForce GTX 1050 Ti || NVIDIA || 555.58.02

### If applicable, add mockups / screenshots to help explain present your vision of the feature


https://github.com/user-attachments/assets/e9ee6d30-7e06-4b7f-8afa-7ad7a338f011



### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

<!-- Click below this line and paste or drag-and-drop your log-->
2024-08-06T09:38:59.318526241+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:42:20.629005008+08:00 [ERROR] XIMClientError: Can't read xim message: Invalid Data ErrorCode: 0
2024-08-06T09:42:44.290444349+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.33213028+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.357638198+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.418349755+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.49233538+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.522232669+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:44.874596303+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:45.65050101+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:45.706520009+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:47.282571217+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:47.38648157+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:47.517322184+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:47.971183452+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:48.010292049+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:48.518148066+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:48.570290212+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:48.602269067+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:48.674549869+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:49.866466067+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:49.899105266+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:49.930246655+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:50.426501205+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:50.465557385+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:50.492365497+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:51.17047822+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:51.226443655+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:52.586206757+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:52.610187114+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:52.644830105+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:55.978299196+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:56.07417037+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:56.338395956+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:56.378833496+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:57.602319521+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.066343695+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.10556975+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.131022156+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.15520299+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.194087116+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.442307325+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.467945043+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.468031715+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.474041726+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.491368794+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.505737864+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.754302452+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.762261667+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.792309575+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.792469108+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.792598782+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:58.816897485+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.07429575+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.107537079+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.107656874+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.10773042+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.107802583+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.129477008+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.146065759+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.418295979+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.457006831+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.56031131+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.589602369+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.610067232+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.626065872+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.922283539+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.957838679+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.957998963+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.958109026+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.962291255+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.985391432+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:42:59.994039192+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.033989726+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.290310165+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.325492636+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.325656303+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.325766855+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.351552937+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.351623793+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.374477125+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.706297101+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.742762683+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.742926051+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.743061572+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.767375872+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:00.791190928+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.218290804+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.260133594+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.260249185+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.260324022+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.282060664+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.290023334+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.330019385+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.650240941+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.691658183+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.691823618+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.691934773+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.698232638+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:01.722009434+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.802093082+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.842638989+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.842812205+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.842923933+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.873521006+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.873610347+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:02.906004411+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.594254003+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.629075627+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.629319589+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.649231138+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.649337394+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:03.665961322+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.330217136+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.362318368+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.362534697+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.386593665+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.38668782+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:04.410473682+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:06.370182509+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:06.450133566+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:08.138149189+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:08.23413341+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:08.274228362+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.042153495+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.076201531+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.082009872+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.10589218+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.834113198+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.865880743+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.866166897+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.891376795+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.897867336+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:09.946108885+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.250100835+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.282867265+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.283029787+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.283139651+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.310324152+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.310395721+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.585790302+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.601863967+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.616238835+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.617860227+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.633178034+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.633856867+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.649885303+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.874119641+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.882014229+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.916217015+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.916381458+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.916446412+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.916553224+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:10.944778985+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.202127977+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.233935153+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.234099958+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.234210057+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.234352787+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.256938407+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.257011141+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.257846124+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.2658183+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.506802342+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.533860979+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.534020236+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.541667787+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.555892082+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.555954862+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.566082157+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.585939493+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.794091822+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.80202518+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.835367361+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.83553041+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.835641219+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.835774318+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:11.861399088+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.066603209+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.085642774+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.089774407+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.099346478+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.105806454+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.105834575+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.121858524+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.137851267+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.571491193+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.585839509+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.601840685+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.617854464+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.633819306+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:12.649806833+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:13.458046444+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:16.641695075+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:16.793985974+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:17.114805004+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:17.297993056+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:19.081890133+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:19.193819074+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:21.346087871+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:21.402008944+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:43:29.763235216+08:00 [INFO] new;
2024-08-06T09:43:29.76682695+08:00 [INFO] keep_updated;
2024-08-06T09:43:29.767024975+08:00 [INFO] new;
2024-08-06T09:43:31.956057446+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:09.529740596+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:11.466530817+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:11.486125137+08:00 [ERROR] XIMClientError: Can't read xim message: Invalid Data ErrorCode: 0
2024-08-06T09:44:22.738547786+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:23.763627739+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:23.763718372+08:00 [ERROR] XIMClientError: Can't read xim message: Invalid Data ErrorCode: 0
2024-08-06T09:44:25.42653406+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:28.681455407+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:28.702926764+08:00 [ERROR] XIMClientError: Can't read xim message: Invalid Data ErrorCode: 0
2024-08-06T09:44:30.970456862+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:32.818382519+08:00 [WARN] Unknown request DestroyIcReply { input_method_id: 1, input_context_id: 21 }
2024-08-06T09:44:32.836287025+08:00 [ERROR] XIMClientError: Can't read xim message: Invalid Data ErrorCode: 0
2024-08-06T09:49:39.44256876+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.47680396+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.477555474+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.491127212+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.515016348+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.53865164+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.851319095+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.877926213+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.883221012+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.891150202+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.913919034+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.922744519+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:39.946808119+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:42.971207272+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:42.996577657+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.003208838+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.01109147+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.034700048+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.066536308+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.098456731+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.394976923+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.418914537+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.426827602+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.445470878+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.458033552+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.458538302+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.474420188+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.506451764+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.79500629+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.81222256+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.826593245+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:43.836637926+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.061173511+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.078305894+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.110735613+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.110850256+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.114072885+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.130155218+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.146333001+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.170123636+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.490458191+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.533378593+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.53353837+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.533647988+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.549729676+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.562157057+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:53.58618232+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.49094268+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.525913581+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.526522237+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.541419479+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.55740613+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.59273756+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:54.609956416+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.762907572+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.7944901+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.802842653+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.818191737+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.834239865+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:56.850236715+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.194882+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.227044467+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.227620026+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.228129674+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.25928826+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.25959257+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.266260142+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.930889618+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:57.97058656+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:58.002371372+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:58.034401547+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.882109291+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.898695544+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.926970282+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.927497219+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.938636026+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:49:59.962458787+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:00.034780068+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.482133134+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.498728591+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.506664083+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.529509965+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.530672746+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.563426527+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.56399411+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:02.590136896+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:50:16.593812323+08:00 [WARN] GPU frame is out of date
2024-08-06T09:50:16.594259711+08:00 [WARN] Acquire failed because the surface is out of date
2024-08-06T09:50:16.597191295+08:00 [INFO] Using surface present mode FIFO_RELAXED
2024-08-06T09:50:16.597563147+08:00 [WARN] Unable to forbid exclusive full screen
2024-08-06T09:50:29.275991903+08:00 [WARN] GPU frame is out of date
2024-08-06T09:50:29.27810795+08:00 [INFO] Using surface present mode FIFO_RELAXED
2024-08-06T09:50:29.278188656+08:00 [WARN] Unable to forbid exclusive full screen
2024-08-06T09:51:25.000731083+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.024461781+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.040591007+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.056642485+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.072665079+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.088713912+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.112648749+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.40917216+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.428792634+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.441058912+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.463085989+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.46479458+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.491452185+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:51:25.504725253+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:52:16.337252361+08:00 [INFO] ignoring macOS specific set_edited
2024-08-06T09:59:19.846590312+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.870656754+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.878629961+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.886555505+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.895910171+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.902604905+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:19.918675424+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.214669886+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.230754582+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.246070092+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.262692321+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.262866749+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.280295956+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.310608507+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.598661298+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.613464369+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.614620971+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.629781177+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.630693702+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.646430485+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.6629332+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.942640489+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.950601375+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.966667154+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.980297301+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.98033769+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.982604068+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:20.998609707+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.014676233+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.286648029+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.297439515+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.302640273+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.314195153+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.318605923+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.330738803+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.347104007+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.374660508+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.622651854+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.630626009+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.647671427+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.647742599+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.654609002+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.664299957+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.664329518+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.680844205+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.878653317+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.897585978+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.897616852+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.902641491+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.914520314+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.914551808+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.918563194+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:21.931623078+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.126647439+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.134591906+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.148085878+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.150631716+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.164651933+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.164683209+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.166566085+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.181552294+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.374609885+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.39836073+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.398602284+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.415161033+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.415192051+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.415217565+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.41522987+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.432748192+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.438631186+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.63061626+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.648223651+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.654583733+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.654614659+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.66508923+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.670611136+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.670647968+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.681662799+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.68656126+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.878606486+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.886561172+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.898548661+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.902564803+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.915394173+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.915424898+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.918571516+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.932257252+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:22.934575454+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.118622681+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.132100077+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.134580208+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.148951494+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.14898285+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.150553131+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.150588372+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.16546242+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.166633778+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.670597937+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.686635675+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.699269189+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.702531871+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.716316775+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.716347491+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.718597987+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.732673785+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.734543424+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:23.750798376+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.28659293+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.302556113+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.31859189+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.33283989+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.334600082+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.349679279+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.366179276+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.390587058+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.718594596+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.73458012+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.750053666+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.751839933+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.751861216+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.766948414+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.766995022+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.783459346+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:24.799641777+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.014542984+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.022523686+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.037449471+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.038498497+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.054016416+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.054524773+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.054590592+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.071164823+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.071227209+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.270531847+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.287866066+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.287928504+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.304619567+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.304680365+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.304701054+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.32139017+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.321451809+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.558482879+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.574485839+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.588320476+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.588361113+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.590499374+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.604601849+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.604641678+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.606510344+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.622578118+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.822563405+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.838564656+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.850673119+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.854520414+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.854570629+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.867903707+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.870518994+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.884089119+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:25.88650078+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.094522465+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.117757445+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.134364787+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.13454477+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.134634562+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.151192633+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.151237908+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.158502923+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.182506032+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.39054529+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.406516003+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.417959565+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.422502235+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.438551834+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.451127017+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.454503151+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:26.502504305+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:27.496280785+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:27.526690785+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:27.574474725+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:28.798455012+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:29.654482172+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:30.054475335+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:30.214472608+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:30.726448143+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:30.766452124+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:30.838416279+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:31.214451879+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:31.278396187+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:31.702411883+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:31.758428615+08:00 [WARN] Unknown button press: ButtonPressEvent { .. }
2024-08-06T09:59:36.794694224+08:00 [ERROR] window not found
2024-08-06T09:59:36.794746923+08:00 [ERROR] window not found
2024-08-06T09:59:36.804173138+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:41.853991374+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:41.858742475+08:00 [ERROR] window not found
2024-08-06T09:59:41.858788717+08:00 [ERROR] window not found
2024-08-06T09:59:46.544141044+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:46.544305973+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:46.544418064+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:46.544445856+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:46.55377187+08:00 [ERROR] window not found
2024-08-06T09:59:46.55380999+08:00 [ERROR] window not found
2024-08-06T09:59:50.067341123+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:50.067526158+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:50.071391975+08:00 [ERROR] window not found
2024-08-06T09:59:50.07142404+08:00 [ERROR] window not found
2024-08-06T09:59:52.987980715+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:52.988100126+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:52.988763948+08:00 [INFO] Using surface present mode FIFO_RELAXED
2024-08-06T09:59:52.988848425+08:00 [WARN] Unable to forbid exclusive full screen
2024-08-06T09:59:56.123583491+08:00 [ERROR] oneshot canceled
2024-08-06T09:59:56.12368344+08:00 [ERROR] oneshot canceled
<!-- Click above this line and paste or drag-and-drop your log--></pre></details>

--- a/crates/gpui/Cargo.toml
+++ b/crates/gpui/Cargo.toml
@@ -143,7 +143,7 @@ xkbcommon = { git = "https://github.com/ConradIrwin/xkbcommon-rs", rev = "fcbb46
     "wayland",
     "x11",
 ] }
-xim = { git = "https://github.com/npmania/xim-rs", rev = "27132caffc5b9bc9c432ca4afad184ab6e7c16af", features = [
+xim = { git = "https://github.com/XDeme1/xim-rs", rev = "d50d461764c2213655cd9cf65a0ea94c70d3c4fd", features = [
     "x11rb-xcb",
     "x11rb-client",
 ] }

--- a/crates/gpui/src/platform/linux/x11/client.rs
+++ b/crates/gpui/src/platform/linux/x11/client.rs
@@ -644,10 +644,9 @@ impl X11Client {
                 window.set_active(true);
                 let mut state = self.0.borrow_mut();
                 state.keyboard_focused_window = Some(event.event);
-                state
-                    .xim_handler
-                    .as_mut()
-                    .map(|handler| handler.window = event.event);
+                if let Some(handler) = state.xim_handler.as_mut() {
+                    handler.window = event.event;
+                }
                 drop(state);
                 self.enable_ime();
             }

--- a/crates/gpui/src/platform/linux/x11/client.rs
+++ b/crates/gpui/src/platform/linux/x11/client.rs
@@ -185,9 +185,7 @@ impl X11ClientStatePtr {
             .build_ic_attributes()
             .push(
                 xim::AttributeName::InputStyle,
-                xim::InputStyle::PREEDIT_CALLBACKS
-                    | xim::InputStyle::STATUS_NOTHING
-                    | xim::InputStyle::PREEDIT_POSITION,
+                xim::InputStyle::PREEDIT_CALLBACKS,
             )
             .push(xim::AttributeName::ClientWindow, xim_handler.window)
             .push(xim::AttributeName::FocusWindow, xim_handler.window)
@@ -555,12 +553,7 @@ impl X11Client {
         let mut xim_handler = state.xim_handler.take().unwrap();
         let mut ic_attributes = ximc
             .build_ic_attributes()
-            .push(
-                AttributeName::InputStyle,
-                InputStyle::PREEDIT_CALLBACKS
-                    | InputStyle::STATUS_NOTHING
-                    | InputStyle::PREEDIT_NONE,
-            )
+            .push(AttributeName::InputStyle, InputStyle::PREEDIT_CALLBACKS)
             .push(AttributeName::ClientWindow, xim_handler.window)
             .push(AttributeName::FocusWindow, xim_handler.window);
 
@@ -588,12 +581,12 @@ impl X11Client {
         state.ximc = Some(ximc);
     }
 
-    pub fn disable_ime(&self) {
+    pub fn reset_ime(&self) {
         let mut state = self.0.borrow_mut();
         state.composing = false;
         if let Some(mut ximc) = state.ximc.take() {
-            let xim_handler = state.xim_handler.as_ref().unwrap();
-            ximc.destroy_ic(xim_handler.im_id, xim_handler.ic_id).ok();
+            let handler = state.xim_handler.as_ref().unwrap();
+            ximc.reset_ic(handler.im_id, handler.ic_id).ok();
             state.ximc = Some(ximc);
         }
     }
@@ -664,7 +657,7 @@ impl X11Client {
                 }
                 state.pre_edit_text.take();
                 drop(state);
-                self.disable_ime();
+                self.reset_ime();
                 window.handle_ime_delete();
             }
             Event::XkbNewKeyboardNotify(_) | Event::MapNotify(_) => {
@@ -823,8 +816,7 @@ impl X11Client {
 
                 if state.composing && state.ximc.is_some() {
                     drop(state);
-                    self.disable_ime();
-                    self.enable_ime();
+                    self.reset_ime();
                     window.handle_ime_unmark();
                     state = self.0.borrow_mut();
                 } else if let Some(text) = state.pre_edit_text.take() {
@@ -974,6 +966,10 @@ impl X11Client {
                 window.set_hovered(true);
                 let mut state = self.0.borrow_mut();
                 state.mouse_focused_window = Some(event.event);
+                state
+                    .xim_handler
+                    .as_mut()
+                    .map(|handler| handler.window = event.event);
             }
             Event::XinputLeave(event) if event.mode == xinput::NotifyMode::NORMAL => {
                 self.0.borrow_mut().scroll_x = None; // Set last scroll to `None` so that a large delta isn't created if scrolling is done outside the window (the valuator is global)
@@ -1082,9 +1078,7 @@ impl X11Client {
                 .build_ic_attributes()
                 .push(
                     xim::AttributeName::InputStyle,
-                    xim::InputStyle::PREEDIT_CALLBACKS
-                        | xim::InputStyle::STATUS_NOTHING
-                        | xim::InputStyle::PREEDIT_POSITION,
+                    xim::InputStyle::PREEDIT_CALLBACKS,
                 )
                 .push(xim::AttributeName::ClientWindow, xim_handler.window)
                 .push(xim::AttributeName::FocusWindow, xim_handler.window)

--- a/crates/gpui/src/platform/linux/x11/client.rs
+++ b/crates/gpui/src/platform/linux/x11/client.rs
@@ -585,8 +585,8 @@ impl X11Client {
         let mut state = self.0.borrow_mut();
         state.composing = false;
         if let Some(mut ximc) = state.ximc.take() {
-            let handler = state.xim_handler.as_ref().unwrap();
-            ximc.reset_ic(handler.im_id, handler.ic_id).ok();
+            let xim_handler = state.xim_handler.as_ref().unwrap();
+            ximc.reset_ic(xim_handler.im_id, xim_handler.ic_id).ok();
             state.ximc = Some(ximc);
         }
     }

--- a/crates/gpui/src/platform/linux/x11/client.rs
+++ b/crates/gpui/src/platform/linux/x11/client.rs
@@ -644,6 +644,10 @@ impl X11Client {
                 window.set_active(true);
                 let mut state = self.0.borrow_mut();
                 state.keyboard_focused_window = Some(event.event);
+                state
+                    .xim_handler
+                    .as_mut()
+                    .map(|handler| handler.window = event.event);
                 drop(state);
                 self.enable_ime();
             }
@@ -966,10 +970,6 @@ impl X11Client {
                 window.set_hovered(true);
                 let mut state = self.0.borrow_mut();
                 state.mouse_focused_window = Some(event.event);
-                state
-                    .xim_handler
-                    .as_mut()
-                    .map(|handler| handler.window = event.event);
             }
             Event::XinputLeave(event) if event.mode == xinput::NotifyMode::NORMAL => {
                 self.0.borrow_mut().scroll_x = None; // Set last scroll to `None` so that a large delta isn't created if scrolling is done outside the window (the valuator is global)

--- a/crates/gpui/src/platform/linux/x11/xim_handler.rs
+++ b/crates/gpui/src/platform/linux/x11/xim_handler.rs
@@ -48,12 +48,7 @@ impl<C: Client<XEvent = xproto::KeyPressEvent>> ClientHandler<C> for XimHandler 
     ) -> Result<(), ClientError> {
         let ic_attributes = client
             .build_ic_attributes()
-            .push(
-                AttributeName::InputStyle,
-                InputStyle::PREEDIT_CALLBACKS
-                    | InputStyle::STATUS_NOTHING
-                    | InputStyle::PREEDIT_NONE,
-            )
+            .push(AttributeName::InputStyle, InputStyle::PREEDIT_CALLBACKS)
             .push(AttributeName::ClientWindow, self.window)
             .push(AttributeName::FocusWindow, self.window)
             .build();
@@ -108,15 +103,6 @@ impl<C: Client<XEvent = xproto::KeyPressEvent>> ClientHandler<C> for XimHandler 
 
     fn handle_close(&mut self, client: &mut C, _input_method_id: u16) -> Result<(), ClientError> {
         client.disconnect()
-    }
-
-    fn handle_destroy_ic(
-        &mut self,
-        client: &mut C,
-        input_method_id: u16,
-        _input_context_id: u16,
-    ) -> Result<(), ClientError> {
-        client.close(input_method_id)
     }
 
     fn handle_preedit_draw(

